client/components/controllers/NotesPhase.js

```diff
@@ -0,0 +1,25 @@
+import React, {Component} from 'react'
+
+class NotesPhase extends Component {
+  handleSubmit = () => {}
+  render() {
+    const {interactionId, pristine, submitting} = this.props
+    return (
+      <div className="row">
+        <form className="col s12" onSubmit={this.handleSubmit}>
+          <div className="row">
+            <div className="input-field col s12">
```

Comment:
Again, if you just want full width things, you can just omit all the 'rows' and 'col s12' stuff.  It will behave like normal HTML blocks and take up the full width.